src/encoded/static/components/biosample_type.js

```diff
@@ -119,11 +120,11 @@ const ListingComponent = ({ context: result, auditIndicators, auditDetail }, rea
             </div>
             <div className="result-item__meta">
                 <div className="type meta-title">Biosample Type</div>
-                {auditIndicators(result.audit, result['@id'], { session: reactContext.session, search: true })}
+                {auditIndicators(result.audit, result['@id'], { session: reactContext.session, sessionProperties: reactContext.session_properties, search: true })}
             </div>
             <PickerActions context={result} />
         </div>
-        {auditDetail(result.audit, result['@id'], { session: reactContext.session, except: result['@id'], forcedEditLink: true })}
+        {auditDetail(result.audit, result['@id'], { session: reactContext.session, sessionProperties: reactContext.session_properties, except: result['@id'], forcedEditLink: true })}
```

Comment:
Like the `except` property, `forcedEditLink` isn’t used anymore, so you can delete this property wherever it occurs if you like.